ansible_runner/utils.py

```diff
@@ -429,3 +429,21 @@ def sanitize_container_name(original_name):
     invalid characters so it can be used in docker/podman CLI commands
     """
     return re.sub('[^a-zA-Z0-9_-]', '_', text_type(original_name))
+
+
+def cli_mounts():
```

Comment:
are we going to remove the other mounts that are listed in runner?

Reply:
With this change only ssh related mounts are added